qpc/tests/cred/conftest.py

```diff
@@ -4,7 +4,8 @@
 
 
 @pytest.fixture(autouse=True)
-def _setup_server_config_file(server_config): ...
+def _setup_server_config_file(server_config):
+    ...
```

Comment:
mmm I liked the oneliner style black had for these. but not enough to keep this from merging  🤷🏽 😆